### PR TITLE
Little fixes (4-2-stable)

### PR DESCRIPTION
**Pull request by korydraughn**
Passed CI.

### CODE REVIEW

**Comment by korydraughn** [comment]
@jasoncoposky  @alanking  Are there any translation issues (throwing -> non-throwing) around the `AGENT_CONN_KW`?

File: server/core/src/client_api_whitelist.cpp [hunk 1]
```diff
@@ -12,6 +12,7 @@ namespace
 {
     auto is_client_to_agent_connection() -> bool
     {
+#if 0
```
Comment (trel) [inline server/core/src/client_api_whitelist.cpp:15]: torch if happy with the #else

File: server/core/src/dataObjOpr.cpp [hunk 2]
```diff
@@ -830,6 +830,7 @@ chkOrphanFile(
     addKeyVal( &genQueryInp.condInput, DISABLE_STRICT_ACL_KW, "disable" );
 
     std::string svr_sid;
+#if 0
```
Comment (trel) [inline server/core/src/dataObjOpr.cpp:833]: same

File: lib/core/include/irods_server_properties.hpp [hunk 3]
```diff
@@ -96,6 +101,10 @@ namespace irods
         configuration_parser config_props_;
     }; // class server_properties
 
+    inline bool server_property_exists(const std::string_view _prop) {
```
Comment (alanking) [inline lib/core/include/irods_server_properties.hpp:104]: Thoughts on the name `server_property_contains`?

File: server/core/src/dataObjOpr.cpp [hunk 4]
```diff
@@ -848,6 +849,22 @@ chkOrphanFile(
             return e.code();
         }
     }
+#else
```
Comment (alanking) [inline server/core/src/dataObjOpr.cpp:852]: I'm assuming the #if 0 block will be deleted in a future commit

**Review by alanking — APPROVED** [review]
The new code looks like it would behave equivalently and is much nicer. :)

File: server/core/src/dataObjOpr.cpp (hunk 4 above)
Comment (korydraughn) [reply server/core/src/dataObjOpr.cpp:852]: the plan was to let the team see the old code beside the new code and verify that they are equivalent.

File: lib/core/include/irods_server_properties.hpp (hunk 3 above)
Comment (alanking) [reply lib/core/include/irods_server_properties.hpp:104]: I was just thinking about our other key-value things lining up with maps terminology. I am fine with this name too

Comment (korydraughn) [reply lib/core/include/irods_server_properties.hpp:104]: i think `exists` works better given the preceding characters. if we use `contains`, i feel `server_property_` needs to be tweaked, else it sounds like `server_property_contains` is asking `does the server property contain _prop`. hopefully that made sense.

Comment (alanking) [reply lib/core/include/irods_server_properties.hpp:104]: Totally

File: server/core/src/client_api_whitelist.cpp (hunk 1 above)
Comment (korydraughn) [reply server/core/src/client_api_whitelist.cpp]: done

File: server/core/src/dataObjOpr.cpp (hunk 2 above)
Comment (korydraughn) [reply server/core/src/dataObjOpr.cpp]: done

**Comment by trel** [comment]
i'm good for #s.

**Comment by korydraughn** [comment]
added #

**Pull request merged**